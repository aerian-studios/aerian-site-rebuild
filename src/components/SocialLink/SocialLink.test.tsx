/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { SocialLink } from "./index";

describe("SocialLink", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <SocialLink
                    className="item"
                    iconName="facebook-f"
                    url="https://www.facebook.com/aerianstudios/"
                />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
