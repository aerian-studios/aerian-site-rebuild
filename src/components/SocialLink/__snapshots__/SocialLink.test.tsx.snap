// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialLink renders correctly 1`] = `
<li
  className=" item"
>
  <a
    href="https://www.facebook.com/aerianstudios/"
    rel="nofollow noreferer"
  />
</li>
`;
